src/components/EditCollectiveDelete.js

```diff
@@ -82,15 +82,8 @@ const DeleteCollective = ({ collective, deleteCollective, deleteUserCollective,
           <FormattedMessage
             values={{ type: collectiveType }}
             id="collective.delete.isNotDeletable-message"
-            defaultMessage={'{type} with transactions, orders or paid expenses cannot be deleted.'}
+            defaultMessage={'{type}s with transactions, orders, or paid expenses cannot be deleted. Please archive it instead.'}
```

Comment:
@Betree sounds like `{type}s` is an incorrect usage of pluralization. How should we handle that? Is that acceptable for now?

Reply:
It is acceptable but we should have proper coverage for pluralization for that kind of strings that we translate very often in the code.

The correct way to do that would be to pass the translation for collective type in another `FormatMessage`, something like:

```jsx
<FormattedMessage
  id="collective.delete.isNotDeletable-message"
  defaultMessage={'{type} with transactions, orders or paid expenses cannot be deleted.'}
  values={{
    type: (
      <FormattedMessage
        id="CollectiveType"
        defaultMessage="{type, select, COLLECTIVE {{count, plural, one {Collective} other {Collectives}}} USER {{{count, plural, one {User} other {Users}}}} }"
        values={{ type, count }}
      />
    ),
  }}
/>
```

Obviously this is verbose and hard to maintain. I would advocate for a `src/components/i18n` folder where we could create generic components for translations that are meant to be shared everywhere so we can abstract that.

For example, we could have a `I18nCollectiveType` component that would turn the code above to:

```jsx
<FormattedMessage
  id="collective.delete.isNotDeletable-message"
  defaultMessage={'{type} with transactions, orders or paid expenses cannot be deleted.'}
  values={{
    type: (
      <I18nCollectiveType type={type} count={count} />
    ),
  }}
/>
```